XenAdmin/Dialogs/HealthCheck/HealthCheckOverviewDialog.Designer.cs

```diff
@@ -141,8 +143,9 @@ private void InitializeComponent()
             // 
             resources.ApplyResources(this.poolDetailsPanel, "poolDetailsPanel");
             this.tableLayoutPanel2.SetColumnSpan(this.poolDetailsPanel, 2);
-            this.poolDetailsPanel.Controls.Add(this.healthCheckStatusPanel, 0, 0);
-            this.poolDetailsPanel.Controls.Add(this.notEnrolledPanel, 0, 1);
+            this.poolDetailsPanel.Controls.Add(this.cannotAlterHealthCheckLabel, 0, 0);

```

Comment:
The label now looks ok at the top. I suggest setting its bottom Margin to 10 to make it stand out a bit more from the sections after it, and maybe add a right and left Padding of 3 to match the other sections.